spec.html

```diff
@@ -4626,7 +4621,7 @@ <h1>ToNumeric ( _value_ )</h1>
 
     <emu-clause id="sec-tonumber" aoid="ToNumber">
       <h1>ToNumber ( _argument_ )</h1>
-      <p>The abstract operation ToNumber converts _argument_ to a value of type Number according to <emu-xref href="#table-11"></emu-xref>:</p>
+      <p>The abstract operation ToNumber takes argument _argument_. It converts _argument_ to a value of type Number according to <emu-xref href="#table-11"></emu-xref>:</p>
```

Comment:
How do you feel about this kind of wording for operations that take a single effectively-unnamed argument?

```suggestion
      <p>The abstract operation ToNumber takes one argument. It converts its argument to a value of type Number according to <emu-xref href="#table-11"></emu-xref>:</p>
```

Reply:
Would does that apply to, say, IsExtensible?

> `
The abstract operation IsExtensible takes argument _O_ (an Object) and returns a completion record which, if its [[Type]] is ~normal~, has a [[Value]] which is a Boolean. It is used to determine whether additional properties can be added to _O_. It performs the following steps:
`

or IsAccessorDescriptor?

>`The abstract operation IsAccessorDescriptor takes argument _Desc_ (a Property Descriptor or *undefined*). It performs the following steps:`

I think I would mildly prefer not to do this just on consistency grounds; it's not obvious to me why we'd omit the name of the argument when there's exactly one but not when there's exactly two, especially in cases where there's no description beyond "it performs the following steps".

Reply:
I think they're fine.

> `The abstract operation IsExtensible takes one argument (an Object) and returns a completion record which, if its [[Type]] is ~normal~, has a [[Value]] which is a Boolean. It is used to determine whether additional properties can be added to its argument. It performs the following steps:`

> `The abstract operation IsAccessorDescriptor takes one argument (a Property Descriptor or *undefined*). It performs the following steps:`

> it's not obvious to me why we'd omit the name of the argument when there's exactly one but not when there's exactly two

We do this all the time in English. When there's no ambiguity, we don't bother to call out things by name. We just use "the" or "it", etc.

Reply:
> When there's no ambiguity, we don't bother to call out things by name. We just use "the" or "it", etc.

My point is that in a lot of cases we _just_ name the arguments and don't later refer to any of them, even as "it".  And in those cases it seems weird to avoid naming them when there's one but not when there's two.

Reply:
Oh I'm fine with not listing them if we don't otherwise refer to them. I assumed every argument would be referred to at least once to describe its type.

Reply:
Many preambles do not currently describe the types of the arguments.

Reply:
I prefer naming all parameters, regardless of the operation's arity.